scripts/android-tracy-runnable-tutorial.sh

```diff
@@ -0,0 +1,238 @@
+  fi
+fi
+
+print_status "Running with the following environment variables:"
+print_status "ANDROID_NDK=${ANDROID_NDK}"
+print_status "IREE_LLVMAOT_LINKER_PATH=${IREE_LLVMAOT_LINKER_PATH}"
+print_status "IREE_ROOT=${IREE_ROOT}"
+print_status "IREE_BUILD_ANDROID=${IREE_BUILD_ANDROID}"
+print_status "TRACY_ROOT=${TRACY_ROOT}"
+print_status "PYTHON_BIN=${PYTHON_BIN}"
+print_status "CC=${CC}"
+print_status "CXX=${CXX}"
+print_status "IREE_INPUT_MLIR=${IREE_INPUT_MLIR}"
+print_status "FUNCTION_INPUTS=${FUNCTION_INPUTS}"
+print_status "ENTRY_FUNCTION=${ENTRY_FUNCTION}"
+
+echo
+
+print_status "Ensuring that we have Tracy source code..."
+if [ ! -d ${TRACY_ROOT}/profiler/build/unix/ ]
+then
+  print_status "Tracy not found at ${TRACY_ROOT}. Either set TRACY_ROOT, or we're going to clone the git repository now."
+  read -p "Press the return key..."
+  git clone https://github.com/wolfpld/tracy "${TRACY_ROOT}"
+fi
+
+echo
+
+print_status "Ensuring that the Tracy profiler is built..."
+if [ ! -x "${TRACY_ROOT}/profiler/build/unix/Tracy-release" ]
+then
+  print_status "Checking Tracy dependencies - assuming Debian."
+  TRACY_DEPS="libcapstone-dev libtbb-dev libglfw3-dev libfreetype6-dev libgtk-3-dev"
+  TRACY_DEPS_COUNT=5
+  TRACY_DEPS_INSTALLED="$(apt list $TRACY_DEPS) 2>/dev/null | grep installed | wc -l"
+  if [ $TRACY_DEPS_INSTALLED != $TRACY_DEPS_COUNT ]
+  then
+    print_status "Installing dependencies now - assuming Debian."
+    sudo apt install $TRACY_DEPS
+  fi
+  make -C "${TRACY_ROOT}/profiler/build/unix" -j12 release
+fi
+
+echo
+
+print_status "Ensuring that we have the input MLIR file..."
+if [ ! -f "${IREE_INPUT_MLIR}" ]
+then
+  print_status "Set IREE_INPUT_MLIR to point to some iree input MLIR file. Not found at current value ${IREE_INPUT_MLIR}."
+  if [ "${IREE_INPUT_MLIR}" == "${DEFAULT_IREE_INPUT_MLIR}" ]
+  then
+    print_status "Okay, we actually know how to generate that file, ${IREE_INPUT_MLIR}, but it will take a while."
+    print_status "Press the return key to continue..."
+    pushd "${IREE_ROOT}"
+    scripts/get_e2e_artifacts.py --test_suites=mobile_bert_squad_tests
+    popd
+  fi
+fi
+
+if [ ! -f "${IREE_INPUT_MLIR}" ]
+then
+  print_status "FATAL: we should have ${IREE_INPUT_MLIR} by that point."
+  exit 1
+fi
+
+echo
+
+print_status "Building IREE for Android in ${IREE_BUILD_ANDROID}..."
+
+mkdir -p "${IREE_BUILD_ANDROID}"
+pushd "${IREE_BUILD_ANDROID}"
+
+cmake -G Ninja ../iree \
+  -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK/build/cmake/android.toolchain.cmake \
+  -DANDROID_ABI="arm64-v8a" \
+  -DANDROID_PLATFORM=android-30 \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DIREE_BUILD_COMPILER=OFF \
+  -DIREE_BUILD_SAMPLES=OFF  \
+  -DIREE_HOST_C_COMPILER=`which "$CC"` \
+  -DIREE_HOST_CXX_COMPILER=`which "$CXX"` \
+  -DIREE_ENABLE_RUNTIME_TRACING=ON
+
+cmake --build .
+
+popd
+echo
+
+print_status "Compiling the input MLIR file into a IREE module..."
+
+IREE_COMPILED_MODULE=/tmp/android_module.fbvm
```

Comment:
```suggestion
IREE_COMPILED_MODULE=/tmp/android_module.vmfb
```

`vmfb` is the preferred extension for these files

Reply:
done